packages/server/README.md

```diff
@@ -78,18 +78,49 @@ Allow `Forwarded`, `X-Forwarded-Proto`, and `X-Forwarded-Host` headers to overri
 Additional headers sent in `Access-Control-Allow-Headers` (`string[]`).
 
 #### `options.generateUrl`
-Control how the upload url is generated (`(req, { proto, host, baseUrl, path, id }) => string)`)
+
+Control how the upload URL is generated (`(req, { proto, host, path, id }) => string)`)
+
+This only changes the upload URL (`Location` header).
+If you also want to change the file name in storage use `namingFunction`.
+Returning `prefix-1234` in `namingFunction` means the `id` argument in `generateUrl` is `prefix-1234`.
+
+```js
+function generateUrl(req, {proto, host, path, id}) {
+  const prefix = getPrefixForUser(req) // your custom logic
+  return `${proto}://${host}${path}/${prefix}/${id}?query=param`
+}
+```
+
+> [!NOTE]
+> `@tus/server` expects everything after the last `/` to be the upload id.
```

Comment:
```suggestion
> `@tus/server` expects everything in the path after the last `/` to be the upload id.
```

This makes it clear that the query string is not interpreted as part of the upload ID. Or is this the case?

Reply:
Good one.

https://github.com/tus/tus-node-server/blob/421edee819f579ceefe85595ce9da23f240cc5bb/packages/server/test/BaseHandler.test.ts#L73-L91